packages/filesystem/src/browser/filesystem-frontend-contribution.ts

```diff
@@ -134,6 +148,22 @@ export class FileSystemFrontendContribution implements FrontendApplicationContri
                 }
             }
         });
+        commands.registerCommand(CommonCommands.NEW_FILE, {
+            execute: async (...args: unknown[]) => {
+                let newFileName = undefined;
```

Comment:
As someone who regularly needs to override methods, I'd really appreciate it if the execution of this command would be in it's own protected method. I know I'm being picky here but I think it makes also for nicer code.

Reply:
Moved to separate protected method